src/renderer/system/archives.js

```diff
@@ -0,0 +1,43 @@
+import ms from 'ms';
+import { lockArchive } from '../../shared/actions/archives';
+import { getAllArchives } from '../../shared/selectors';
+
+const __cache = {
+  timer: null
+};
+
+export const setupArchiveActions = store => ({
+  lockAllArchives() {
+    const archives = getAllArchives(store.getState());
+    if (archives) {
+      archives.forEach(
+        ({ id, status }) =>
+          status === 'unlocked' ? store.dispatch(lockArchive(id)) : ''
+      );
+    }
+  },
+  lockArchiveTimer() {
```

Comment:
This should be called when there's activity, too. It shouldn't lock while in focus if there's activity going on.